tools/SetupFlow/DevHome.SetupFlow/Views/ReviewView.xaml

```diff
@@ -37,6 +37,7 @@
     <setupFlowBehaviors:SetupFlowNavigationBehavior.ContentTemplate>
         <CheckBox
             x:Uid="ms-resource:///DevHome.SetupFlow/Resources/Review_ReadAndAgree"
+            Visibility="{x:Bind ViewModel.RequiresTermsAgreement, Mode=OneWay, Converter={StaticResource BoolToVisibilityConverter}}"

```

Comment:
Just to record this somewhere... this is documented here: https://learn.microsoft.com/en-us/windows/uwp/xaml-platform/x-bind-markup-extension#:~:text=Starting%20in%20Windows%2010%2C%20version%201607%2C%20the,target%20versions%2C%20see%20Version%20adaptive%20code.

Min version with automatic visibility converting is 1607. WinUI 3 only goes down to [version 1809](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/#:~:text=app%20on%20Windows%2011%20and%20downlevel%20to%20Windows%2010%2C-,version%201809,-.). So we're always safe.

Reply:
Awesome, thank you for sharing the link! Pushed the changes 🙌

Reply:
note that the min version comment is about system xaml (uwp), because that's the OS release that the converter was added. WinUI 3 was forked after that, so it has that converter code. The converter will work on all versions where WinUI 3 runs.